doc/chapters/design/index.rst

```diff
@@ -0,0 +1,39 @@
+SoftwareContainer Design
+************************
+
+SoftwareContainer has been designed as a building block enabling sandboxing of application contents in a larger system.
```

Comment:
We don't have any guidelines for that AFAIK. Perhaps we should talk pros and cons but this commit is at least consistent with the rest of the user docs.